webapp-django/crashstats/crashstats/static/crashstats/css/daily.less

```diff
@@ -5,6 +5,10 @@ h1 {
     font-size: 1.90em;
     font-weight: bold;
 }
+.page-heading p.new-daily-report-link {
+    float: right;
+}
+
```

Comment:
Why not use the same layout that I've used on Signature report / report/list and TCBS? That is, just putting the link right next to the page title. There's even less markup needed. 

Note that I don't mind if you prefer your solution over mine and change how it's done in other places, I just care about consistency. It's good to have patterns that users can reproduce across the website.